<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDialog</class>
 <widget class="QDialog" name="HistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="queryDataButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>851</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>851</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QDateTimeEdit" name="startTimeEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="endTimeEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="tankComboBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="prePushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pageLineEdit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>450</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="allPageNumberLabel">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>480</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoPushButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>跳到</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
